endpoints/users/controller.py

```diff
@@ -91,30 +93,41 @@ def post(self):
                 return {"message": "User not registered"}, 404
             if bcrypt.checkpw(credentials['password'].encode('utf8'), user['password'].encode('utf8')) == False:
                 return {"message": "Password does not match"}, 422
+
             auth_token = create_access_token(identity=user['id'])
-            return {"access_token":auth_token, "message":f"logged in as {user.username}"}, 200
+            return {"access_token": auth_token, "message": f"logged in as {user['username']}"}, 200
 
         except Exception as error:
-            response.errorResponse(str(error))
-            return response.__dict__
+            print(error)
+            return {'message': 'Something went wrong', 'timestamp': round(time.time())}, 500
+
 
 class LogoutResource(Resource):
+    @jwt_required
     def post(self):
         try:
             jti = get_raw_jwt()['jti']
-            revoked_token = RevokedToken(jti = jti)
+            revoked_token = RevokedToken(jti=jti)
             revoked_token.add()
             return {'message': 'Access token has been revoked'}
```

Comment:
I don't think you need to return an actual message here. Just a status code 200

Reply:
Done, thanks